Makefile

```diff
@@ -9,6 +9,7 @@ XGB_IMG ?= xgbserver:latest
 PYTORCH_IMG ?= pytorchserver:latest
 ALIBI_IMG ?= alibi-explainer:latest
 STORAGE_INIT_IMG ?= storage-initializer:latest
+CRD_OPTIONS ?= "crd:trivialVersions=true"
```

Comment:
Not sure what this does?

Reply:
https://book.kubebuilder.io/reference/generating-crd.html#multiple-versions